SPCTR-3445: Fix - Lottie width issue inside container (#3546)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Lottie - Resolved an issue in which the SVG aligment would not work as intended in the TwentyTwentyThree theme.
* Fix: Global - Resolved an issue wherein a block error would occur when attempting to select typography font families in the presence of multiple iFrames.
* Fix: Global - Resolved an issue wherein adding Containers, Forms, Lottie blocks and the Popup Builder caused errors when the Gutenberg plugin was active.

> * Fix: Lottie - Resolved an issue in which the width of block is getting adopted from container.

Reviewer: ```suggestion
* Fix: Lottie - Resolved an issue in which the width of the block would be adopted from the parent Container.
```

Author: updated.
